apollo-runtime/src/main/java/com/apollographql/apollo/internal/field/CacheFieldValueResolver.java

```diff
@@ -38,7 +40,7 @@ public CacheFieldValueResolver(ReadableCache readableCache, Operation.Variables
     }
   }
 
-  private Record valueFor(Record record, Field.ObjectField field) {
+  private Record valueFor(Record record, Field.ObjectField field) throws IOException {
```

Comment:
IOException or ApolloException? -- I choose IOException because that is what our ResponseReader currently expects.